x-pack/plugins/fleet/public/applications/integrations/sections/epm/screens/detail/index.scss

```diff
@@ -3,3 +3,9 @@
 .fleet__epm__shiftNavTabs {
   margin-left: $euiSize * 6 + $euiSizeXL * 2 + $euiSizeL;
 }
+
+@media (max-width: 767px) {
+  .header_container .euiFlexGroup--responsive > .euiFlexItem {
```

Comment:
I've not got loads of CSS experience but it is this selector quite fragile? It's a shame we can't easily add tests for screen size incase a change to the eui flexgroup component broke this. What is the margin without this style? is this an issue with the component at small screen sizes?